docs/home/howto/MODULES.md

```diff
@@ -2,99 +2,76 @@
+* faster and more focused release cycle.
+
+Those and multiple other factors (eg.: readability, reliability) influence this direction of changes.
 
 ## User point of view
 
-From user point of view there will be no significant changes in the nearest future as it will be still possible to install Epiphany "classic way" so with single `epicli` configuration using whole codebase as monolith. 
+From a user point of view, there will be no significant changes in the nearest future as it will be still possible to install Epiphany "classic way" so with a single `epicli` configuration using a whole codebase as a monolith.
+
+For those who want to play with new features, or will need newly introduced possibilities, there will be a short transition period which we consider as a kind of "preview stage". In this period there will be a need to run each module separately by hand in the following order:
 
-For those who want to play with new features, or will need newly introduced possibilities, there will be short transition period which we consider as kind of "preview stage". In this period there will be need to run each module separately by hand in following order: 
- * moduleA init
- * moduleA plan
- * moduleA apply
- * moduleB init
- * moduleB plan
- * moduleB apply
- * ...
- 
-Init, plan and apply phases explanation you'll find in next sections of this document. Main point is that dependent modules have to be executed one after another during this what we called "preview stage". Later, with next releases there will be separate mechanism introduced to orchestrate modules dependencies and their consecutive execution. 
+* moduleA init
+* moduleA plan
+* moduleA apply
+* moduleB init
+* moduleB plan
+* moduleB apply
+* ...
+
+Init, plan and apply phases explanation you'll find in next sections of this document. Main point is that dependent modules have to be executed one after another during this what we called "preview stage". Later, with next releases there will be separate mechanism introduced to orchestrate modules dependencies and their consecutive execution.
 
 ## New scenarios
 
-In 0.8 we introduce possibility to use AKS or EKS as kubernetes providers. That is introduced with modules mechanism, so we introduced first four modules: 
- * [Azure Basic Infrastructure](https://github.com/epiphany-platform/m-azure-basic-infrastructure) (AzBI) module
- * [Azure AKS](https://github.com/epiphany-platform/m-azure-kubernetes-service) (AzKS) module
- * [AWS Basic Infrastructure](https://github.com/epiphany-platform/m-aws-basic-infrastructure) (AwsBI) module
- * [AWS EKS](https://github.com/epiphany-platform/m-aws-kubernetes-service) (AwsKS) module
- 
-Those 4 modules with the Classic Epiphany used with `any` provider allow change of on-prem kubernetes cluster with managed kubernetes services. 
+In 0.8 we introduce the possibility to use AKS or EKS as Kubernetes providers. That is introduced with modules mechanism, so we introduced the first four modules:
```

Comment:
introduce synonyms, maybe like this?
In 0.8 we offer the possibility to use AKS or EKS as Kubernetes providers. That is introduced with modules mechanism, so we launched the first four modules:

Reply:
:+1: